source/includes/fact-get-help-menu.rst

```diff
@@ -0,0 +1,4 @@
+View links for documentation, contact support, community forums, 
```

Comment:
[suggestion] We want to make these list items parallel (same part of speech). Also looks like we missed "and" here.
```suggestion
View links for documentation, support, community forums, and
```